cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/BuildCommand.java

```diff
@@ -182,6 +185,10 @@ public void execute() {
                 BuilderUtils.compileAndWriteJar(sourceRootPath, pkgName, targetFileName, buildCompiledPkg,
                         offline, lockEnabled, skiptests, experimentalFlag, dumpBIR);
             } else {
+                // Load the configuration file. If no config file is given then the default config file i.e.
+                // "ballerina.conf" in the source root path is taken.
+                LauncherUtils.loadConfigurations(sourceRootPath, configFilePath);
```

Comment:
Doesn't this mean that we won't be able to give a config file to the build command if the target is the JVM?

Reply:
We should allow it. Fixed with 0071468. Thanks :)